app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :likes
   has_many :comments
+
+  def update_posts_counter
+    Post.find_by(:id, post_id).count.update
+  end
+
+  def last_5_comments
+    Comment.last(5)
+  end
```

Comment:
- [ ] It will show you `5 last comments` but not in descending order. For that, you have to include one more condition.

Hint: `comments.order('created_at Desc').limit(5)`

Reply:
Thank you very much, @Dipeshtwis :pray:
Please, on your hint, am I using `comments.order...` or `Comment.order...`?
I just want to be sure. Thanks for your time.

Reply:
I would recommend using `comments`. We generally use class name in controllers and not in models.

Reply:
Thank you, again

Reply:
This issue is well resolved! ✔️